.php_cs.dist

```diff
@@ -26,5 +26,6 @@ return Symfony\CS\Config\Config::create()
         Symfony\CS\Finder\DefaultFinder::create()
             ->exclude('Symfony/CS/Tests/Fixtures')
             ->in(__DIR__)
+            ->exclude('build')
```

Comment:
@keradus phpunit stores the code coverage in the ``build`` folder on Travis. These files should not be fixed